src/exchanges/crdtExchange.ts

```diff
@@ -381,6 +381,10 @@ class CrdtManager {
   }
 
   sendMutations() {
+    if (this.#mutations.size === 0) {
```

Comment:
Kinda curious under what conditions this can occur. If the mutations are empty and we're being called, that means we didn't unsubscribe. Should we also be unsubscribing here?

Reply:
I just kept repeatedly getting errors in my UI that were really annoying and  put this here to get them out of my way. This really doesn't belong in this PR, but should be addressed in a separate PR.